src/App/ComponentsDocumentation/components/Select/constants.js

```diff
@@ -15,17 +15,24 @@ export const selectOverview = {
                         title: "Label add on",
                         inputs: [
                             {
-                                id: "help_icon",
-                                name: "Help icon",
+                                id: "optional",
+                                name: "Is optional",
                                 value: {
-                                    tooltip: true
+                                    optional: true
                                 }
                             },
                             {
-                                id: "optional",
-                                name: "Is optional",
+                                id: "hint_text_select",
+                                name: "Hint text",
                                 value: {
-                                    optional: true
+                                    helpBlock: "Hint text"
+                                }
+                            },
+                            {
+                                id: "expander_hint_text",
+                                name: "Expander hint",
+                                value: {
+                                    expandingHintText: "Expanding hint text",
                                 }
                             }
```

Comment:
As we discussed :)

![image](https://user-images.githubusercontent.com/44967773/164182940-a5df25b5-d02b-4c11-baa8-3781f427c1da.png)